datalake/src/main/java/com/sequenceiq/datalake/service/resize/recovery/ResizeRecoveryService.java

```diff
@@ -90,6 +96,13 @@ public SdxRecoveryResponse triggerRecovery(SdxCluster sdxCluster, SdxRecoveryReq
             switch (actualStatusForSdx.getStatus()) {
                 case STOP_FAILED:
                     return new SdxRecoveryResponse(sdxReactorFlowManager.triggerSdxStartFlow(sdxCluster));
+                case STOPPED:
+                    if (sdxCluster.isDetached()) {
+                        //return new SdxRecoveryResponse(sdxReactorFlowManager.triggerSdxResizeRecovery(sdxCluster, null));
+                        throw new NotImplementedException("Cluster is currently in an unrecoverable state");
```

Comment:
this contradicts with line#75 where the validity check returns RECOVERABLE

Reply:
@yg-gd is this the same as line #75 right?

Reply:
correct now Non-Recoverable we shouldn't run into this